fastapi_sso/sso/gitlab.py

```diff
@@ -14,12 +16,33 @@ class GitlabSSO(SSOBase):
     provider = "gitlab"
     scope = ["read_user", "openid", "profile"]
     additional_headers = {"accept": "application/json"}
+    base_endpoint_url = "https://gitlab.com"
+
+    def __init__(
+        self,
+        client_id: str,
+        client_secret: str,
+        base_endpoint_url: Optional[Union[pydantic.AnyHttpUrl, str]] = None,
```

Comment:
Could you please move the new argument to the end of the argument list? Also, I'm not super sure about the `pydantic.AnyHttpUrl`, we are not using the url in any pydantic model, so this feel a little bit odd, what do you think?

Reply:
Yes, I'll move the param to end. 

As for `pydantic.AnyHttpUrl`, I was just focusing on the current implementation and added this so that the code would be in the style of the project.

Examples where `AnyHttpUrl` is used:
- [base](https://github.com/tomasvotava/fastapi-sso/blob/master/fastapi_sso/sso/base.py#L66)
- [microsoft](https://github.com/tomasvotava/fastapi-sso/blob/master/fastapi_sso/sso/microsoft.py#L25)

Reply:
Oh, I haven't even noticed it's already there, then no problem, use it here as well 👌 Thanks!